tests/typegen/evaluation/test_file_type_hints_collector.py

```diff
@@ -6,96 +6,146 @@
 from typegen.evaluation import FileTypeHintsCollector
 
 root = pathlib.Path.cwd()
-relative_sample_folder_path = pathlib.Path(
-    "tests") / "resource" / "typegen" / "evaluation" / "file_type_hints_collector"
+relative_sample_folder_path = (
+    pathlib.Path("tests")
+    / "resource"
+    / "typegen"
+    / "evaluation"
+    / "file_type_hints_collector"
+)
 sample_folder_path = root / relative_sample_folder_path
-sample_data_folder_path = root / "tests" / "resource" / "external" / "PyTypes_BinaryFiles" / "sample_typehint_data_files"
+sample_data_folder_path = (
+    root
+    / "tests"
+    / "resource"
+    / "external"
+    / "PyTypes_BinaryFiles"
+    / "sample_typehint_data_files"
+)
 
 
 def test_file_type_hints_collector_returns_correct_data_for_filename():
     filename1 = str(relative_sample_folder_path / "file_with_type_hints.py")
     filename2 = "file_with_type_hints.py"
 
     test_object = FileTypeHintsCollector()
-    method_to_tests_to_test = [lambda: test_object.collect_data_from_file(root, filename1),
-                               lambda: test_object.collect_data_from_file(sample_folder_path, filename2)]
+    method_to_tests_to_test = [
```

Comment:
I think I accidentally named this wrongly :) Please change all occurences in this file to methods_to_test.

Reply:
Err, I don't really want to change your test code. Please fix this after I have fixed your other comments